src/Session/commands.py

```diff
@@ -127,8 +127,16 @@ def open(self, foodStorage_type):
         # If food storage exists with this name, prints open success message. 
         print(OPEN_FOOD_STORAGE_SUCCESS_MESSAGE.format(foodStorage_type, foodStorage_name))
                    
+    ''' Allows user to save their profile information. '''
```

Comment:
Should include that it saves it to a json file under the profiles folder

Reply:
Done!